<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>675</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Autonomous Driving</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QPlainTextEdit" name="te_main"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="lb_top">
      <property name="text">
       <string>TextLabel</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pb_send">
      <property name="text">
       <string>Send</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuGeneral">
    <property name="title">
     <string>General</string>
    </property>
    <addaction name="actionPreferences"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <widget class="QMenu" name="menuConnection">
    <property name="title">
     <string>Connection</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
   </widget>
   <addaction name="menuGeneral"/>
   <addaction name="menuFile"/>
   <addaction name="menuConnection"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuit">
   <property name="icon">
    <iconset theme="application-exit">
     <normaloff>../../../../../../../home/alex/.designer/backup</normaloff>../../../../../../../home/alex/.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset theme="preferences">
     <normaloff>../../../../../../../home/alex/.designer/backup</normaloff>../../../../../../../home/alex/.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Preferences</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open">
     <normaloff>../../../../../../../home/alex/.designer/backup</normaloff>../../../../../../../home/alex/.designer/backup</iconset>
   </property>
   <property name="text">
    <string>Open Map XML</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Disconnect</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionQuit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
